.travis.yml

```
@@ -2,4 +2,4 @@ language: node_js
 node_js:
   - "0.10"
 script:
-  "node tests/tests.js"
+  "npm i && npm test"
```

```suggestion
  "npm test"
```
`npm install` is already done by default in travis on the "install" step, before "script"